src/interpreter/InterpretAdditionalItems.ts

```diff
@@ -12,7 +12,7 @@ import { Interpreter, InterpreterOptions } from './Interpreter';
  */
 export default function interpretAdditionalItems(schema: Schema, model: CommonModel, interpreter : Interpreter, interpreterOptions: InterpreterOptions = Interpreter.defaultInterpreterOptions): void {
   if (model.type?.includes('array') === false) {return;}
-  const additionalItemsModel = interpreter.interpret(schema.additionalItems || true, interpreterOptions);
+  const additionalItemsModel = interpreter.interpret(schema.additionalItems === undefined ? true : schema.additionalItems, interpreterOptions);
```

Comment:
Not better `if (schema.additionalItems === false) {return;}` ?

Reply:
I find it a bit weird behavior tbh, to precheck the schema before trying to interpret it since if the interpreter encounter a false schema it returns undefined. Making the process more dynamic and more natural IMO 🙂

Reply:
Why weird? 😄 If someone is knows the JSON Schema when see the `schema.additionalItems === false` he/she knows then the next code will perform actions only on `undefined`, `true` or on object.

Reply:
IMHO

```ts
if (additionalItems === false) {return;}  const additionalItemsModel = interpreter.interpret(schema.additionalItems || true, interpreterOptions);
```

is more readable

Reply:
It's weird because it should not be the sub-interpreters job to figure out how to interpret a schema | boolean, that's the job of the main interpret 🙂

Reply:
Let it be